src/components/FreeflowingCard/index.jsx

```diff
@@ -1,26 +1,32 @@
 import PropTypes from "prop-types";
-import {
-    Card,
-    CardBody,
-} from "@nextui-org/react";
+import { Card, CardBody } from "@nextui-org/react";
+import { useNavigate } from "react-router-dom";
 import { PercentageScore } from "../PercentageScore";
 import "./style.css";
 
-export const FreeflowingCard = ({cardName}) => {
-    const currentCardName = cardName;
+export const FreeflowingCard = ({ cardName, cardUrl }) => {
+  const navigateTo = useNavigate();
 
-    return (
-        <Card className="card-div">
-                  <CardBody>
-                    {/* <div className="card-div"> */}
-                    <PercentageScore />
-                <p className="card-name">{currentCardName}</p>
-                    {/* </div> */}
-                  </CardBody>
-                </Card>
-    )
-}
+  const checkIfLoggedIn = () => {
+    if (localStorage.getItem("jobQuestEmail")) {
+      navigateTo(`/${cardUrl}`); //navigate to taskCard
```

Comment:
Okay, this makes sense. I was wondering how the routing to the Task component within App.css would be done now that we have 6 different endpoints that lead to the one component. @khairahscorner 

Also, correct me if I'm wrong, I would get the resource name from the URL and parse information rendered based on that correct? Thank you!

Reply:
Nevermind, I now see your comments on my previous pr!

Reply:
Yes, correct! And well done with the page, you handled the logic really well👍

Reply:
Thank you!